src/locales/en/translationEn.json

```diff
@@ -493,6 +493,7 @@
         "memberOf": "Member of",
         "location": "Location",
         "edit": "Edit",
+        "Edit": "Edit",
```

Comment:
Any reason why there are two edits? "edit" and "Edit".
If it's required, this is ambiguous, right?

Reply:
Please try to follow standard naming.

Reply:
This was from my end.This happened during a hotfix.I have the retained "edit": "Edit", for both french and english translation.